TimberAPIExample/ThunderstorePackage/TimberAPIExample/plugins/specifications/BuildingSpecification.Bakery.Folktails.json

```diff
@@ -0,0 +1,28 @@
+{
+	"BuildingId": "Bakery.Folktails",
```

Comment:
Nice, varied, examples

Reply:
As mentioned in another comment idea to remove the BUildingId and just find it by the specification name instead?

Reply:
I like that idea